modules/storage.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] In this project, students are expected to migrate classes into functions using ES6 arrow syntax. Please make the necessary changes accordingly.